subworkflows/local/euk_mags_generation.nf

```diff
@@ -163,10 +163,16 @@ workflow EUK_MAGS_GENERATION {
     quality = CONCATENATE_QUALITY_FILES.out.concatenated_result
 
     // -- qs50 -- //
+    empty_result_concoct = EUKCC_CONCOCT.out.eukcc_csv.map{ meta, csv -> return tuple(meta, []) }
```

Comment:
```suggestion 
    // EUKCC may return empty files, but the downstream jobs require
    // all of the samples to be present (with empty [] to mark empty outputs)
    // because the results may be empty we are constructing empty results
    // using the csv mapping files, we need the meta argument for each sample 
    empty_result_concoct = EUKCC_CONCOCT.out.eukcc_csv.map{ meta, csv -> return tuple(meta, []) }
```

Reply:
The code is clear enough, but I would add one comment there to make it easier in the future to understand it